stepconf/stepconf_test.go

```diff
@@ -34,6 +33,8 @@ var valid = map[string]string{
 	"file":          "/etc/hosts",
 	"dir":           "/tmp",
 	"export_method": "dev",
+	"emptyptr":      "",
```

Comment:
Can we add test cases for unset integer? Disallowing ptr for string may make sense if it us not useful.